### DISCUSSION TITLE
Connecting to a remote mongo instance

### DISCUSSION THREAD

**Discussion by AzielioTor** [keystonejs/keystone#5449] [Questions] [answered]
Hi everyone,

I recently inherited a keystone application and have been trying to set it up locally myself.

I am currently running into an issue where keystone is looking for the mongo db to be on localhost/127.0.0.1, but I'm hosting it on a remote server and for the life of me cannot find any connection string in the application, or documentation on how to change it to connect to a remote db. 

Any assistance on the matter would be greatly appreciated!

**Comment by bladey** [comment, marked as answer]
Hi AzielioTor, this discussion forum is only used for feature requests for Keystone Next at this time, could you please post your question on [Stackoverflow or Slack](https://github.com/keystonejs/keystone/issues/new/choose) with more information such as the version of Keystone you're using and we'd be happy to help out! I'll keep an eye out for the question.
